unstructured/partition/pptx.py

```diff
@@ -41,15 +42,13 @@ def partition_pptx(
         metadata.
     """
 
-    if not any([filename, file]):
-        raise ValueError("One of filename or file must be specified.")
+    # Verify that only one of the arguments was provided
+    exactly_one([filename, file], ["filename", "file"])
 
-    if filename is not None and not file:
+    if filename:
```

Comment:
```suggestion
    if filename is not None:
```
Similar to my other comment, if `filename == ""` and `file` is `None`, it will bypass all the clauses in the `if` statement and `presentation` will be undefined in line 56.